src/track/beatfactory.cpp

```diff
@@ -1,36 +1,34 @@
-#include <QtDebug>
+#include "track/beatfactory.h"
+
 #include <QStringList>
+#include <QtDebug>
 
-#include "track/beatgrid.h"
-#include "track/beatmap.h"
-#include "track/beatfactory.h"
+#include "track/beats.h"
 #include "track/beatutils.h"
+#include "track/track.h"
 
-BeatsPointer BeatFactory::loadBeatsFromByteArray(const Track& track,
-                                                 QString beatsVersion,
-                                                 QString beatsSubVersion,
-                                                 const QByteArray& beatsSerialized) {
+mixxx::BeatsPointer BeatFactory::loadBeatsFromByteArray(const TrackPointer track,
+        QString beatsVersion,
+        QString beatsSubVersion,
+        const QByteArray& beatsSerialized) {
     if (beatsVersion == BEAT_GRID_1_VERSION ||
         beatsVersion == BEAT_GRID_2_VERSION) {
+        /*
```

Comment:
Either delete the code or add a comment why it is commented out and needs to be kept.

Reply:
This part of code is still work in progress. Will disappear at some point.